packages/sdk/src/ZSshClient.ts

```diff
@@ -58,7 +58,9 @@ export class ZSshClient extends AbstractRpcClient implements Disposable {
                 Logger.getAppLogger().debug("Client disconnected");
                 opts.onClose?.();
             });
-            client.mSshClient.connect(ZSshUtils.buildSshConfig(session));
+            client.mSshClient.connect(
+                ZSshUtils.buildSshConfig(session, opts.keepAliveInterval ? opts.keepAliveInterval * 1000 : 30e3),
```

Comment:
I guess 0 is the same as 30 seconds 😋

Reply:
Oops, its not supposed to be - 0 should be passed thru to the `ssh2` package to disable the keep-alive messages, this is a good catch 😋

Reply:
Resolved in 3814430a1959756c9d2385815dc3f7d168155d26